### PR TITLE
Add model & schema & resolver for Reaction

### PR DESCRIPTION
**Pull request by daadaadaah**
Description
--------------------------------------------------------------------------------------------------------------
 - [x] Add model and schema for Reaction
 - [x] Add resolver for Reaction 
 - [ ] Emoji test in client
 - [ ] Add test code for Reaction.

Related Issues
--------------------------------------------------------------------------------------------------------------
 #37

Tests
--------------------------------------------------------------------------------------------------------------
No test for now.

### CODE REVIEW

File: schemas/schema.graphql [hunk 1]
```diff
@@ -20,6 +21,7 @@ type Query {
   channels: [Channel!]!
   friends: [User!]!
   galleries(userId: String!): [Gallery!]!
+
```
Comment (hyochan) [inline schemas/schema.graphql]: I think you can remove the extra line here.

**Comment by hyochan** [comment]
I think you can start working on the resolvers!

File: schemas/schema.graphql (hunk 1 above)
Comment (daadaadaah) [reply schemas/schema.graphql]: ok!

**Comment by daadaadaah** [comment]
# Sunday(3/1) Meetup Feedback
- Use "yarn migrate:generate" When make file for migration.

**Comment by daadaadaah** [comment]
Closes via #76

**Comment by daadaadaah** [comment]
Closes via #71

**Comment by daadaadaah** [comment]
closes #76

**Pull request closed without merging**